docs/features/models/squad.rst

```diff
@@ -188,7 +165,7 @@ We got 57.88 EM and 65.91 F-1 on ground truth Wikipedia article (we used the sam
 +---------------+-----------------------------------------------+----------------+-----------------+
 | Model config                                                  |    EM (dev)    |    F-1 (dev)    |
 +===============================================================+================+=================+
-| :config:`DeepPavlov <squad/multi_squad_noans.json>`           |     57.88      |     65.91       |
```

Comment:
Maybe keep reference to the old model results for comparison?
upd link to previous DeepPavlov version